src/WpBookingsFrontUi.php

```diff
@@ -93,7 +97,13 @@ public function setup()
              * @since [*next-version*]
              */
             'eddbk_front_app_holder_template' => function (ContainerInterface $c) {
-                return $this->_makeTemplate('application-holder.html');
+                return new JsonStringPlaceholderTemplate(
+                    $this->_getTemplateContent('application-holder.html'),
+                    $c->get('bookings_front_ui/templates_config/token_start'),
+                    $c->get('bookings_front_ui/templates_config/token_end'),
+                    $c->get('bookings_front_ui/templates_config/token_default'),
+                    $this->_normalizeArray($c->get('bookings_front_ui/wizard_template/context_json_strings'))
```

Comment:
You don't need an array here, but a list. Try [`_normalizeIterable()`][]. Let me know if you need me to explain why.

[`_normalizeIterable()`]: https://github.com/Dhii/normalization-helper-base/blob/develop/src/NormalizeIterableCapableTrait.php#L31